src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -15,30 +15,52 @@
 import java.util.function.BiConsumer;
 import java.util.stream.Collectors;
 
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
 public class TransactionDao {
 
-    private static final String FIND_TRANSACTION_BY_EXTERNAL_ID = "SELECT * FROM transaction " +
-            "WHERE external_id = :externalId " +
-            "AND (:gatewayAccountId is NULL OR gateway_account_id = :gatewayAccountId)";
+    private static final String FIND_TRANSACTION_BY_EXTERNAL_ID =
+            "SELECT t.*, po.paid_out_date AS paid_out_date FROM transaction t " +
+            "LEFT OUTER JOIN payout po on " +
+            "t.gateway_payout_id = po.gateway_payout_id " +
+            ":payoutAccountId " +
+            "WHERE t.external_id = :externalId " +
+            "AND (:gatewayAccountId is NULL OR t.gateway_account_id = :gatewayAccountId)";
 
-    private static final String FIND_TRANSACTION_BY_EXTERNAL_ID_AND_GATEWAY_ACCOUNT_ID = "SELECT * FROM transaction " +
-            "WHERE external_id = :externalId " +
-            "AND gateway_account_id = :gatewayAccountId " +
+    private static final String FIND_TRANSACTION_BY_EXTERNAL_ID_AND_GATEWAY_ACCOUNT_ID =
```

Comment:
For this query gateway_account_id is always available. We do not need conditional replacement